api/services/Websockets.js

```diff
@@ -9,14 +11,26 @@ const validMessageTypes = [
   'newPost'
 ]
 
+export function broadcast (room, messageType, payload, socketToExclude) {
```

Comment:
i thought i left a comment here about reworking this so that it doesn't create a new connection on every call... guess i never submitted it? anyway it doesn't have to block the merge, just noting it.

Reply:
although if you wanted to take a stab at this it would be welcome. should be pretty simple, just like
```js
var io

export function broadcast () {
  if (io) {
    io = emitter(process.env.REDIS_URL)
    io.redis.on('error', err => {
      rollbar.error(err, null, {room, messageType, payload})
    })
  }
}
```